charts/hathor-fullnode/templates/statefulset.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "hathor-full-node.fullname" . }}
+  labels:
+    {{- include "hathor-full-node.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.statefulset.replicas }}
+  serviceName: {{ include "hathor-full-node.fullname" . }}
+  selector:
+    matchLabels:
+      {{- include "hathor-full-node.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "hathor-full-node.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "hathor-full-node.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: generate-peer-id
```

Comment:
If we send the peer-id as parameter we don't need to run this command, right?

Reply:
Do you think the Chart user should provide a peer-id when installing the chart?

I can make this configurable, what do you think?
People would choose if they want to provide a peer-id, or let it be generated automatically.

Reply:
We have aligned in Slack that this could be left for future improvements.